MIGRATION.md

```diff
@@ -49,6 +49,17 @@ The easiest to replace were `assert` (which was replaced by simple conditions &
 
 Lastly, the `buffer` polyfill turned out to be the trickiest to remove, resulting in the largest number of breaking changes. Since the `Buffer` object is not native to the browser all apis were migrated to the superclass of `Buffer` → `Uint8Array`s. For a detailed write up of why we and many libraries are choosing to make this transition, check out this [blog post](https://sindresorhus.com/blog/goodbye-nodejs-buffer) by Sindre Sorhus.
 
+List of all replaced polyfills that can be removed from your webpack.config.js / vite.config.js / other bundling config files:
```

Comment:
Having helped others or looked at their code these polyfills cannot always be safely removed espefially if they are using other libraries that require them.  You might also mention `events` which used to have to be specified in the vite config

Reply:
Added along with a note that these may still be needed for other libraries / code written.